### DISCUSSION TITLE
About the Accuracy criteria

### DISCUSSION THREAD

**Discussion by EricGaoOm** [deepmodeling/dpgen#1601] [Q&A] [answered]
As a new user of dpgen, I am wondering during each iteration, what citeria of key value decide the exit of a loop? Or in another word, whether the dpgen have parameter like EDIFF and EDIFFG in vasp?

**Comment by njzjz** [comment, marked as answer]
Exiting the loop is impossible until all iterations defined in `model_devi_jobs` are completed. However, the training process can be skipped if the number of new structures is fewer than `fp_task_min`.
